### ISSUE TITLE
How do I get the foreclosure price of a property?

### ISSUE THREAD

**Issue by cue232s** [synewaves/rubillow#23]
Looking or retirieve the foreclosorsure price of a property. 
Using Rubillow::Property.deep_search_results(address: address, citstatezip: zipcode).price 
returns the zillow zestimate it seems. 

Where as using : Rubillow::PropertyDetails.updated_property_details zpid: zipid) give me the following error: 

```
 NoMethodError: undefined method 'text' for nil:NilClass 
from /Users/eduerondu/.rvm/gems/ruby-2.0.0-p0/gems/rubillow-0.0.8/lib/rubillow/models/updated_property_details.rb:68:in `parse'
```

**Comment by synewaves** [comment]
I don't believe the foreclosure price was available in the Zillow API when this library was created.  Can you give an example of the XML that would contain the foreclosure price?

**Comment by bkubic** [comment]
I get the same error when trying to use the updated_property_details API - any chance this can be corrected?
